lib/src/replay/element_parsers/element_data.dart

```diff
@@ -16,24 +16,20 @@ class ElementData {
     children?.add(elementData);
   }
 
-  List<Rect> extractRects([bool isRoot = true]) {
-    List<Rect> rects = [];
-
-    if (!isRoot) {
-      rects.add(rect);
-    }
+  List<ElementData> extractRects([bool isRoot = true]) {
```

Comment:
```suggestion
  List<ElementData> extractRects({bool isRoot = true}) {
```

I think optionals are better than positionals